kudo-operator/docs/latest/submission.md

```diff
@@ -1,2 +1,160 @@
 Submitting Spark Applications
----
\ No newline at end of file
+---
+
+In order to deploy your application to Kubernetes using Kudo Spark Operator, first you need to describe it as a Kubernetes object. To do that, you need to create a specification in `yaml` format, that will contain all the configuration required for the application.
```

Comment:
let's add a prerequisites section explicitly mentioning that KUDO Spark must be installed